adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/impl/LoraProtocolAdapter.java

```diff
@@ -265,8 +265,8 @@ private void doUpload(final RoutingContext ctx, final Device device, final Strin
         }
     }
 
-    private JsonObject createDownstreamJson(final String base64Payload) {
-        return new JsonObject().put("payload", base64Payload);
+    private JsonObject createDownstreamJson(final String base64Payload, final JsonObject originalMessage) {
+        return new JsonObject().put("payload", base64Payload).put("originalMessage", originalMessage);
```

Comment:
can you please define the property names as constants, e.g. `FIELD_BINARY_PAYLOAD` and `FIELD_ORIG_UPLINK_MESSAGE`? We also usually do not use camel case for JSON properties but use a dash to separate parts, e.g. `orig-message` instead of `origMessage`.

Reply:
Done